tests/views_integration/Tribe/Events/Views/Full_Site/Archive_EventsTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Tribe\Events\Editor\Full_Site;
+
+use Spatie\Snapshots\MatchesSnapshots;
+use TEC\Events\Editor\Full_Site\Archive_Block_Template;
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlTestCase;
+
+class Archive_EventsTest extends HtmlTestCase {
+	use MatchesSnapshots;
+
+	public function setUp(): void {
+		parent::setUp();
+		add_filter( 'tribe_editor_should_load_blocks', '__return_true', PHP_INT_MAX );
+		tribe( \Tribe__Events__Editor__Provider::class )->register();
+	}
+
+	/**
+	 * Sorts and removed dynamic fields for consistent structures.
+	 *
+	 * @param $template
+	 *
+	 * @return array
+	 */
+	public static function normalize_wp_template( $template ): array {
+		$array_template = (array) $template;
+		asort( $array_template );
+		// Dynamic fields, ditch them.
+		foreach ( [ 'wp_id', 'author' ] as $field ) {
+			$array_template[ $field ] = null;
+		}
```

Comment:
Can we just `unset()` them?
```suggestion
		// Dynamic fields, ditch them.
		unset( $array_template[ wp_id ]);
		unset( $array_template[ author ]);
```
...or will that create issues down the line?

Reply:
I'd just prefer showing all the fields that ought to be there for parity, but it's half dozen of one 6 of the other. Makes it easier to see a null value that shouldn't be null than locating a missing field.